### ISSUE TITLE
Hosted pyFLAMEGPU docs via CI

### ISSUE THREAD

**Issue by ptheywood** [FLAMEGPU/FLAMEGPU2-userguide#9]
Currenlty C++ API docs are generated and hosted by CI in this repo, but this does not include any docs for pyflamegpu. 

This is dependent upon the pyflamegpu docs being possible on the main repo in the first place (https://github.com/FLAMEGPU/FLAMEGPU2/issues/455)

It will also probably require the `pyflamegpu` target to be built (or possibly a sub target) and therfore require CUDA to be installed on the CI (without very significant changes to CMake)

**Comment by Robadob** [comment]
Yeah might want to create a separate CMake target, that calls swig to gen the huge py file, but doesn't call the compiler.
